russian/thinking-about/pen-paper/pen-paper.md

```diff
@@ -0,0 +1,24 @@
+Ты развернул бумажку и прочитал:
+
+"
+-Когда ты чувствуешь, что тебе трудно встать утром, подумай про себя: я пробудился для исполнения обязанностей человека, почему же огорчаться тем, что мне нужно сделать то, для чего я родился, для чего я послам, в мир? 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.